app/gimletlet/app.toml

```diff
@@ -72,13 +27,49 @@ start = true
 # task-slots is explicitly empty: packrat should not send IPCs!
 task-slots = []
 
-[tasks.user_leds]
-name = "drv-user-leds"
-features = ["stm32h7"]
+[caboose]
+tasks = ["control_plane_agent", "caboose_reader"]
+region = "flash"
+size = 256
+default = true
```

Comment:
This should go in the common TOML since having a caboose on all images is nice even if it rarely gets updated